<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="64.448" tests="4" failures="1">
  <testsuite name="Root Suite" timestamp="2025-06-17T09:57:20" tests="0" file="cypress\e2e\tests\LoginTest.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Success and Fail login flow" timestamp="2025-06-17T09:57:20" tests="4" time="64.434" failures="1">
    <testcase name="Success and Fail login flow should login successfully with valid credentials" time="16.834" classname="should login successfully with valid credentials">
      <failure message="Timed out retrying after 5000ms: Expected to find content: &apos;header.overview&apos; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 5000ms: Expected to find content: 'header.overview' but never did.
    at Context.eval (webpack://cypress-framework/./cypress/e2e/tests/LoginTest.cy.js:32:29)]]></failure>
    </testcase>
    <testcase name="Success and Fail login flow should fail to login with invalid credentials" time="11.294" classname="should fail to login with invalid credentials">
    </testcase>
    <testcase name="Success and Fail login flow should fail to login for No credentials " time="9.485" classname="should fail to login for No credentials ">
    </testcase>
    <testcase name="Success and Fail login flow should perform login and logout" time="10.698" classname="should perform login and logout">
    </testcase>
  </testsuite>
</testsuites>